core/src/main/scala/org/apache/spark/deploy/master/WorkerInfo.scala

```diff
@@ -107,4 +107,6 @@ private[spark] class WorkerInfo(
   def setState(state: WorkerState.Value): Unit = {
     this.state = state
   }
+
+  def isAlive(): Boolean = this.state == WorkerState.ALIVE
```

Comment:
Hi Andrew 

Base on same reason on previous. I think best way to do that is add a one line method on WorkerInfo

How do you think?